src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1683,10 +1683,31 @@
   <data name="DetectPreviewFeaturesTitle" xml:space="preserve">
     <value>This API requires opting into preview features</value>
   </data>
-  <data name="ImplementsEmptyPreviewInterfaceMessage" xml:space="preserve">
-    <value>'{0}' implements a preview interface. See https://aka.ms/dotnet-warnings/preview-features for more information.</value>
-  </data>
   <data name="StaticAndAbstractRequiresPreviewFeatures" xml:space="preserve">
     <value>Using both 'static' and 'abstract' modifiers requires opting into preview features. See https://aka.ms/dotnet-warnings/preview-features for more information.</value>
   </data>
+  <data name="DerivesFromPreviewClassMessage" xml:space="preserve">
+    <value>'{0}' derives from preview class '{1}' and therefore needs to opt into preview features. See https://aka.ms/dotnet-warnings/preview-features for more information.</value>
+  </data>
```

Comment:
Note: All analyzers in this package get a help link:

https://github.com/dotnet/roslyn-analyzers/blob/a497be39fc800a3436287e9e0f1987c9284ed27d/src/Utilities/Compiler/DiagnosticDescriptorHelper.cs#L36

The analyzer should eventually be documented in dotnet/docs. So likely we won't need the link in the message itself.

Reply:
Right, each diagnostic has its doc page, i thought the URL here for some kind of conceptual for preview feature

Reply:
> I thought the URL here for some kind of conceptual for preview feature

I do think the conceptual documentation link is justified here--it can be a document that describes Preview Features, their level of support, etc. The diagnostic id help link can point to the standard documentation about the _analyzer_ itself.

/cc @gewarren